src/components/Markdown/MetaData.tsx

```diff
@@ -8,12 +8,13 @@ interface MetaDataProps {
 }
 
 const MetaData = ({ title, description, date }: MetaDataProps) => {
+	const formattedTitle = "Kastel - " + title;
```

Comment:
this isn't needed, just do `Kastel - {title}` in the title field

Reply:
it's also for og:description

Reply:
(guessing you mean for the og:title?) In that case use a template literal instead

```suggestion
	const formattedTitle = `Kastel -  ${title`;
```